src/audio/src/CMakeLists.txt

```diff
@@ -1,9 +1,31 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
-add_local_sources(sof src_generic.c src_hifi2ep.c src_hifi3.c src_hifi4.c src_hifi5.c src_common.c src.c)
+set(base_files src_generic.c src_hifi2ep.c src_hifi3.c src_hifi4.c src_hifi5.c src_common.c src.c)
```

Comment:
I know this is a copy of existing, but we should not need to build hifi2 for hifi 5 targets

Reply:
Can be fixed later though. @singalsu fyi.